### PR TITLE
Include JPiOS in issue template

### PR DESCRIPTION
**Pull request by twstokes**
Adds JPiOS to the version collected for the issue. I assume most of the time these versions will match, but it will also serve as a reminder to include both versions if they differ.

### CODE REVIEW

**Comment by twstokes** [comment]
Open to ideas and suggestions!

File: .github/ISSUE_TEMPLATE.md
Comment (mokagio) [inline .github/ISSUE_TEMPLATE.md]: What do you think of using full names for the apps? We might have gotten away with shortening WordPress to WP because the context of the repo made it self evident what we meant, but I do wonder if outside contributors would immediately recognize "JP" as a shortcut for Jetpack.

```suggestion
##### Tested on [device], iOS [version], Jetpack iOS / WordPress iOS [version]
```

**Review by mokagio — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md
Comment (twstokes) [reply .github/ISSUE_TEMPLATE.md]: SGTM!

**Comment by mokagio** [comment]
@twstokes thank you for this refinement 🙌 

It's annoying that we have to wait for CI before merging this... _One day_™ our checks will be smarter. I enabled auto-merge the loop on this can close automatically.

**Comment by twstokes** [comment]
Not sure why auto-merge didn't do its thing. Merging now.

**Pull request merged**
